pages/common/select.md

```diff
@@ -17,4 +17,4 @@
 
 - Create a menu from a Bash array:
 
-`{{fruits}}=({{apple orange pear banana}}); select {{word}} in ${{{fruits[@]}}}; do echo ${{word}}; done`
+`{{fruits}}={{(apple orange pear banana)}}; select {{word}} in ${{{fruits[@]}}}; do echo ${{word}}; done`
```

Comment:
Dammit I had a brain fart. The parenthesis must be outside the placeholder because it's strictly specified to be an array.